### ISSUE TITLE
Proton Update Changes Spacing in URL Bar

### ISSUE THREAD

**Issue by xelaseer** [jannispinter/indicatetls#51]
Thanks for the great extension.

It appears that Firefox's new Proton UI refresh has new address bar styling, including spacing changes.

This affects this extension's URL bar indicator, making it arguably look a little unintentional and distorted post-update, as can be seen in the below screenshot. Specifically, it appears the indicator spacing is now taller and thinner.

[Screenshot](https://postimg.cc/rKxjfQbc)

Note that this new UI refresh can be turned on by setting browser.proton.urlbar.enabled (this only controls the URL bar, other proton prefs enable other parts of the theme) to true in Nightly in about:config.

While this is not enabled by default yet, it appears that it will make its way to release reasonably soon.

**Comment by jannispinter** [comment]
Thanks for pointing this out, as far as I can tell from the screenshot, the problems are that

- the pageAction icon is now stretched (taller than wide)
- the detection of the theme (dark theme, bright theme) does not work

Our current detection for the dark theme is quite hacky, we just check for the RGB color:
`    if (themeInfo.colors && themeInfo.colors.icons === "rgb(249, 249, 250, 0.7)") {`
This can be fixed by implementing a proper theme detection, or just change the RGB color values and continue to use the hack.

Why the page action is stretched is not clear to me yet, it looks like it is a bug in the current Firefox nightly, as the spec clearly says that the pageAction icon should be at a 1:1 aspect ratio. We simply use `browser.pageAction.setIcon` to set the icon. Does this affect other extensions as well?

**Comment by xelaseer** [comment]
Yeah, this does seem to effect other extensions like FlagFox and True Sight as well.

Should I file a bug with Mozilla?

Just a piece of info regarding my theme colour, I am using the Dracula Dark Theme, so the colours you see in my screenshot may not be representative of the official dark theme.

**Comment by xelaseer** [comment]
This screenshot shows the colour of the official dark theme in Proton, as it is currently.

It does appear to be under heavy development though.

[Screenshot](https://postimg.cc/PpdfxfgR)

**Comment by xelaseer** [comment]
Looks like a bug has been filed with Mozilla here:
https://bugzilla.mozilla.org/show_bug.cgi?id=1696628
